sites/partners/lib/helpers.ts

```diff
@@ -120,3 +119,28 @@ export function arrayToFormOptions<T>(arr: T[], label: string, value: string): F
     value: val[value],
   }))
 }
+
+/**
+ * Create Date object with date and time which comes from the TimeField component
+ */
+export const createTime = (
```

Comment:
This (maybe?) looks the same as the `formatTimeFieldToDate` function in `TimeField`?

Reply:
It's a little bit different, but I've noticed that it's not used currently.